contracts/JBMultiTerminal.sol

```diff
@@ -328,76 +338,77 @@ contract JBMultiTerminal is JBOperatable, Ownable, IJBMultiTerminal {
+    ) external virtual override returns (uint256 netLeftoverPayoutAmount) {
+        return _sendPayoutsOf(_projectId, _token, _amount, _currency, _minReturnedTokens);
     }
 
-    /// @notice Allows a project to send funds from its overflow up to the preconfigured allowance.
-    /// @dev Only a project's owner or a designated operator can use its allowance.
-    /// @dev Incurs the protocol fee.
-    /// @param _projectId The ID of the project to use the allowance of.
-    /// @param _token The token being distributed. This terminal ignores this property since it only manages one token.
-    /// @param _amount The amount of terminal tokens to use from this project's current allowance, as a fixed point number with the same amount of decimals as this terminal.
-    /// @param _currency The expected currency of the amount being distributed. Must match the project's current funding cycle's overflow allowance currency.
-    /// @param _minReturnedTokens The minimum number of tokens that the `_amount` should be valued at in terms of this terminal's currency, as a fixed point number with 18 decimals.
-    /// @param _beneficiary The address to send the funds to.
+    /// @notice Allows a project to pay out funds from its surplus up to the current surplus allowance.
+    /// @dev Only a project's owner or an operator with the `JBPermissionIds.USE_ALLOWANCE` permission from that owner can use the surplus allowance.
+    /// @dev Incurs the protocol fee unless the caller is a feeless address.
+    /// @param _projectId The ID of the project to use the surplus allowance of.
+    /// @param _token The token being paid out from the surplus.
+    /// @param _amount The amount of terminal tokens to use from the project's current surplus allowance, as a fixed point number with the same amount of decimals as this terminal.
+    /// @param _currency The expected currency of the amount being paid out. Must match the currency of one of the project's current ruleset's surplus allowances.
+    /// @param _minTokensUsed The minimum number of terminal tokens that should be used from the surplus allowance (including fees), as a fixed point number with 18 decimals. If the amount of surplus used would be less than this amount, the transaction is reverted.
+    /// @param _beneficiary The address to send the surplus funds to.
     /// @param _memo A memo to pass along to the emitted event.
-    /// @return netDistributedAmount The amount of tokens that was distributed to the beneficiary, as a fixed point number with the same amount of decimals as the terminal.
+    /// @return netAmountPaidOut The number of tokens that were sent to the beneficiary, as a fixed point number with the same amount of decimals as the terminal.
     function useAllowanceOf(
         uint256 _projectId,
         address _token,
         uint256 _amount,
         uint256 _currency,
-        uint256 _minReturnedTokens,
+        uint256 _minTokensUsed,
```

Comment:
`used` is an odd verb here, this is the minimum number of tokens the user should receive. Maybe something like  `_minTokensReceived` or `_minTokenAmount`

Reply:
Changed to `minTokensPaidOut`. Thank you!